tuda_workspace_scripts/robots.py

```diff
@@ -158,6 +168,21 @@ def _load_pc_from_yaml(
     )
 
 
+def _load_zenoh_router_from_yaml(config: dict[str, Any]) -> ZenohRouter:
+    if "address" not in config:
+        raise ValueError(f"Address not specified for router {config}")
+    address = config["address"]
+    if "port" in config:
```

Comment:
You could use `port = config.get("port", 7447)` here to be save some lines and make it more readable.